cpu/riscv/extensions/BUILD

```diff
@@ -0,0 +1,972 @@
+# Constraint settings describing RISC-V extensions.
+licenses(["notice"])
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
```

Comment:
It feels like this should have a description of the intended usage. That you create a platform from the riscv base and add in the settings that are relevant to the particular silicon you have.
That is, I want to capture the design choice somewhere to serve as documentation.